### PR TITLE
Adding `x mistral` module introduction in README.md

### PR DESCRIPTION
**Pull request by edwinjhlee**
https://github.com/mistralai/client-python/issues/89#issuecomment-2283383068

### CODE REVIEW

**Comment by pandora-s-git** [comment]
Hi @edwinjhlee , could you provide a notebook or a markdown+script explaining how to use it and PR to the cookbook repo? Then we can point towards it in the readme!

We would like to have all cookbooks in this repo! Thank you! <3

**Comment by edwinjhlee** [comment]
> Hi @edwinjhlee , could you provide a notebook or a markdown+script explaining how to use it and PR to the cookbook repo? Then we can point towards it in the readme!
> 
> We would like to have all cookbooks in this repo! Thank you! <3

Is the example and code in the https://www.x-cmd.com/mod/mistral#examples sufficient enough ? 

image

**Comment by edwinjhlee** [comment]
OK. Now I see your points. I will write a short markdown documents and submit it to this repo.

**Comment by pandora-s-git** [comment]
Thank you! The most detailed and practical the best! If possible to make a notebook would be great, if not a markdown file would be nice indeed!

**Comment by sophiamyang** [comment]
Hi @edwinjhlee, thanks so much for the PR. Could you change your README.ipynb to README.md? I think a markdown file makes more sense here. Thanks!

File: README.md [hunk 1]
```diff
@@ -72,3 +72,4 @@ Disclaimer: Examples contributed by the community and partners do not represent
 | [Streamlit Integration - Chat with PDF](third_party/streamlit/README.md)                                        | UI chat, demo, RAG     | Streamlit  |
 | [SubQuestionQueryEngine.ipynb](third_party/LlamaIndex/RouterQueryEngine.ipynb)                                  | agent                  | LLamaIndex |
 | [LLM Judge: Detecting hallucinations in language models](third_party/wandb/README.md)                           | fine-tuning, evaluation | Weights & Biases |
+| [`x mistrial`: CLI & TUI APP Module in X-CMD](third_party/x-cmd/README.ipynb)                                   | CLI, TUI APP, Chat     | x-cmd |
```
[the remark this answers is not in the document]
Comment (edwinjhlee) [reply README.md:75]: I am so so sorry ...

We will fix it right away.

**Comment by edwinjhlee** [comment]
> Hi @edwinjhlee, thanks so much for the PR. Could you change your README.ipynb to README.md? I think a markdown file makes more sense here. Thanks!

OK. We will convert it to markdown.

**Review by pandora-s-git — APPROVED** [review]

**Comment by pandora-s-git** [comment]
Looks good! cc @sophiamyang merging?

**Comment by edwinjhlee** [comment]
> Looks good! cc @sophiamyang merging?

Sorry... We are just consider refactoring the layout.

**Comment by pandora-s-git** [comment]
oh no problem we can wait

**Comment by edwinjhlee** [comment]
> oh no problem we can wait

Hello, we've finished making the changes. Would you mind taking a look?

**Review by pandora-s-git — APPROVED** [review]

**Comment by pandora-s-git** [comment]
Looks good for me! Will be merging this soon! Thank you so much!

**Pull request merged**